### PR TITLE
Support normalizing in lasso and enet

### PR DESCRIPTION
**Pull request by agorshk**
Support normalizing in lasso and elastic-net (add preprocessing and update coefficients)

### CODE REVIEW

**Comment by PetrovKP** [comment]
Need to check performance. For other versions - 0.21, 0.22, these changes must also be made. Right?

File: daal4py/sklearn/linear_model/_coordinate_descent_0_23.py [hunk 1]
```diff
@@ -89,13 +68,45 @@ def _daal4py_fit_enet(self, X, y_, check_input):
...
+    self._y = y
+
     penalty_L1 = np.asarray(self.alpha*self.l1_ratio, dtype=X.dtype)
     penalty_L2 = np.asarray(self.alpha*(1.0 - self.l1_ratio), dtype=X.dtype)
     if (penalty_L1.size != 1 or penalty_L2.size != 1):
         raise ValueError("alpha or l1_ratio length is wrong")
     penalty_L1 = penalty_L1.reshape((1,-1))
     penalty_L2 = penalty_L2.reshape((1,-1))
 
+    #normalizing and centering
```
Comment (PetrovKP) [inline daal4py/sklearn/linear_model/_coordinate_descent_0_23.py:82]: Ok now, but this part have a lot of a time. It is necessary to understand the popularity of this case and maybe expand the daal's interfaces, it is also possible for linear/ridge regression

File: daal4py/sklearn/linear_model/_coordinate_descent_0_23.py [hunk 2]
```diff
@@ -245,6 +290,12 @@ def _daal4py_fit_lasso(self, X, y_, check_input):
     #set coef_ and intersept_ results
     lasso_model = lasso_res.model
     self.daal_model_ = lasso_model
+
+    #update coefficients if normalizing and centering
+    if self.fit_intercept and self.normalize:
+        lasso_model.Beta[:,1:] = lasso_model.Beta[:,1:] / X_scale
```
Comment (PetrovKP) [inline daal4py/sklearn/linear_model/_coordinate_descent_0_23.py:296]: `X_scale` is vector? You div vector on vector. Right?

File: daal4py/sklearn/linear_model/_coordinate_descent_0_23.py (hunk 1 above)
Comment (agorshk) [reply daal4py/sklearn/linear_model/_coordinate_descent_0_23.py:82]: yes, it is a temporary solution

**Comment by agorshk** [comment]
> Need to check performance. For other versions - 0.21, 0.22, these changes must also be made. Right?

yes, it was fixed for Sklearn version 0.21-0.24, performance measurement in a separate letter

**Pull request merged**